code/datums/ruins.dm

```diff
@@ -0,0 +1,22 @@
+/datum/map_template/ruin
+	//name = "A Chest of Doubloons"
+	name = null
```

Comment:
Sorry I should've said what I meant by this;
If you use `name` for the function of `suffix` you can instead do `mappath = prefix + name + ".dmm"` or `mappath = "[prefix][name[.dmm"`
Obviously the only upshot is this is saving a single var, but it probably counts in the long run.